types/react-big-calendar/lib/addons/dragAndDrop.d.ts

```diff
@@ -9,7 +9,8 @@ export interface EventInteractionArgs<TEvent> {
     event: TEvent;
     start: stringOrDate;
     end: stringOrDate;
-    isAllDay: boolean;
+    isAllDay?: boolean;
+    resourceId: string | number;
```

Comment:
This should be updated to match other places where resourceId prop is already in the type
```suggestion
    resourceId?: string | number;
```